contracts.ts

```diff
@@ -0,0 +1,104 @@
+import * as Config from '@taqueria/protocol/Config';
+import * as Contract from '@taqueria/protocol/Contract';
+import * as i18n from '@taqueria/protocol/i18n';
+import * as LoadedConfig from '@taqueria/protocol/LoadedConfig';
+import * as SanitiziedAbsPath from '@taqueria/protocol/SanitizedAbsPath';
+import * as SanitizedArgs from '@taqueria/protocol/SanitizedArgs';
+import * as SHA256 from '@taqueria/protocol/SHA256';
+import * as TaqError from '@taqueria/protocol/TaqError';
+import { attemptP, chain, map, reject, resolve } from 'fluture';
+import { pipe } from 'https://deno.land/x/fun@v1.0.0/fns.ts';
+import { isEmpty, omit, toPairs } from 'rambda';
+import { getConfig } from './taqueria-config.ts';
+import { joinPaths, readTextFile, writeJsonFile } from './taqueria-utils/taqueria-utils.ts';
+
+type contractRow = {
+	'Name': string;
+	'Last Known Hash': string;
+	'Source file': string;
+};
+
+const hasContracts = (config: LoadedConfig.t) => isEmpty(config.contracts);
```

Comment:
Isn't it supposed to be `isNotEmpty` or something like that? Since we are testing if the contracts folder has content?